sse2neon.h

```diff
@@ -3026,7 +3104,7 @@ FORCE_INLINE __m128i _mm_castps_si128(__m128 a)
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_castsi128_pd
 FORCE_INLINE __m128d _mm_castsi128_pd(__m128i a)
 {
-#if defined(__aarch64__)
+#if defined(__aarch64__) || defined(_M_ARM64)
     return vreinterpretq_m128d_f64(vreinterpretq_f64_m128i(a));
 #else
     return vreinterpretq_m128d_f32(vreinterpretq_f32_m128i(a));
```

Comment:
Again, I'm placing a comment on an unrelated line due to GitHub limitations...

Just below this, _mm_clflush() is currently doing nothing on MSVC (with another FIXME: MSVC comment).

Reply:
ACK, will use the implementation from `compiler-rt`: https://codebrowser.dev/llvm/compiler-rt/lib/builtins/clear_cache.c.html#66